rts/lra/coordinator/src/main/java/io/narayana/lra/coordinator/api/Coordinator.java

```diff
@@ -700,8 +701,9 @@ private URI toURI(String lraId) {
         try {
             return url.toURI();
         } catch (URISyntaxException e) {
-            LRALogger.i18nLogger.error_invalidStringFormatOfUrl(lraId, e);
-            String errorMsg = lraId + ": Invalid format of LRA id URL format to convert to URI " + e.getMessage();
+            String logMsg = LRALogger.i18nLogger.error_invalidStringFormatOfUrl(lraId, e);
+            LRALogger.logger.error(logMsg);
+            String errorMsg = String.format("%s (%s)", logMsg, e.getMessage());
```

Comment:
logMsg already contains the error message. Just use logMsg.
You have done the same in other places so I will not repeat this comment but it needs changing throughout.